output/certs/bc_ssai.csv

```diff
@@ -0,0 +1,70 @@
+key_algorithm_oid,test_result
```

Comment:
I'm pretty sure the `output` directory isn't supposed to be checked in since it gets recreated on the automated test runs.  Possibly you want to add a .gitignore containing `output`.

Reply:
Hi Daniel, thanks for the review. Noted on the `output` directory. I have removed it and added output/ to .gitignore. Kindly request to merge this PR.